docs/assets/diagrams/transfer/02-create-transaction.puml

```diff
@@ -0,0 +1,132 @@
+@startuml
+
+title Create Transaction
+
+hide footbox
+
+autonumber
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  database Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+Alice -> App: I want to make this transaction
+
+rnote right of App #Light 
+  ""createDoc(""
+  ""  "transactions/users/1234/<uuid>",""
+  ""  {""
+  ""    payee: {""
+  ""      phoneNumber: "+1-222-222-2222"""
```

Comment:
Do you want to include the type i.e. MSISDN?

Reply:
updated